src/collections/events/docker-developer-community-meetup-2022/index.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: "Composing Cloud Native Infrastructure with Docker Desktop and Meshery"
+date: 2022-06-11
+thumbnail: ./dockerBengaluru.webp
+eurl: https://www.meetup.com/Docker-Bangalore/events/285342797/
+type: Meetups
+event-category: talk
+published: true
+featured: true
+product: Meshery Extensions
+resource: true
+upcoming: true
+# Deck: N/A
+# Video: N/A
+---
+
+import { Link } from "gatsby";
+
+<p>
+  Join Layer5, In-person at <a href="https://www.meetup.com/Docker-Bangalore/events/285342797/">Docker Developer Community Meetup - Bengaluru</a> as
+  we share on the latest Meshery Docker Extension and how you can manage your cloud native applications using it.
+</p>
+
+<br />
+
+<h2>
+  Talk:{" "}
+  <a href="https://www.meetup.com/Docker-Bangalore/events/285342797/">Composing Cloud Native Infrastructure with Docker Desktop and Meshery</a>
+</h2>
+
+<br />
+
+<p>
+
+The Meshery Docker Extension’s ability to import Docker Compose apps, convert them to Kubernetes applications, and deploy them on any service mesh is a powerful enabler for microservices developers, who need to develop, test, and deploy their modern applications in the context of and compatibility with any service mesh. Along with this, the extension can help you to,
+
+<ul>
+<li>Get service mesh support for your Docker Compose apps: Import your Docker Compose apps. Configure and deploy them to Kubernetes and any service mesh</li>
+<li>Help you with single-click deployment of any service mesh: Supports 10 different service meshes to the fingertips of developers in connection with Docker Desktop’s ability to deliver Kubernetes locally.</li>
+<li>Detection of Kubernetes environments: Scan your kubeconfigs and select your current Kubernetes environment. Switch from one environment to another one</li>
+</ul>
+
+This talk can help the attendees to improve upon their K8s or docker deployments as well as help them in managing different service meshes simultaneously. I have a pretty basic knowledge of cloud-native stuff having worked with Kubernetes, Docker, Helm, and using some of the service meshes, trying them out for different stuff or applications. I have been using docker desktop for the past year and a half and I’m also a maintainer of Meshery and have contributed to several other open-source projects.
+
+</p>
+
+Speakers:
+
+<ul>
+  <li>
+    <Link to="/community/members/adithya-krishna">Aditya Krishna Sharma</Link>
+  </li>
+  <li>
+    <Link to="https://linkedin.com/in/uravgkarthik">Karthik Ravishankar</Link>
```

Comment:
General note: `<Link>` is for internal URLs only. `<a>` is used for external URLs.
Specific rule: Only links to community member profiles are allowed here. It'd be great to have a link to Karthik's social profile. It's something that he'll need to earn in the community first, though.